lib/tasks/creator_seed.rake

```diff
@@ -1,17 +1,69 @@
 # frozen_string_literal: true
 require "csv"
-namespace :vocab do
+namespace :creators do
+  desc "Collects creator names and saves them to csv"
+  task collect: :environment do
+    creators_path = Rails.root.join("data", "creators.csv")
+
+    solr = Blacklight.default_index.connection
+
+    response = solr.get 'select', params: { "facet.field": ["creator_sim"], "facet.limit": -1 }
+    creators_array = response["facet_counts"]["facet_fields"]["creator_sim"]
+
+    CSV.open(creators_path, 'w', write_headers: true, headers: ["display_name", "result_count"]) do |row|
+      creators_array.each_slice(2) do |pair|
+        row << pair
+      end
+    end
+    # Same as above, one-line syntax
+    # CSV.open(creators_path,'w', :write_headers=> true, :headers => ["display_name", "result_count"]) { |row| creators_array.each_slice(2) { |pair| row << pair } }
```

Comment:
Maybe take out the one-line option.  If we really need it, I think we can get back there easy enough, but I think the multi-line version is much more readable.